configure

```diff
@@ -97,7 +97,7 @@ check_toolchain()
         EMACS=""
     else
         emacs_version=$($EMACS --version 2>/dev/null | head -n 1)
-        if echo $emacs_version | grep -Eq 'GNU Emacs 2[6789]'; then
+        if echo $emacs_version | grep -Eq 'GNU Emacs (2[6789]|30)'; then
```

Comment:
Let's make this future-proof - something like `3[0-9]` instead of `30`

Reply:
Is the intent to just support any version newer than 26? Maybe in that case we should just parse the version properly?

Reply:
Yeah, basically. Parsing it properly is fine as well :)

Reply:
Tried my take at doing a proper(ish) version check.